org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -33,10 +33,10 @@
                     "url": "https://static.rust-lang.org/dist/rust-1.48.0-armv7-unknown-linux-gnueabihf.tar.gz",
                     "sha256": "3aed4a63ebdd57690a31d11afbe95e6407edc224a6769be5694a1ed43bf899cb",
                     "x-checker-data": {
-                        "type": "html",
-                        "url": "https://www.rust-lang.org/",
-                        "version-pattern": "Version ((?:\\d+\\.)?(?:\\d+\\.)?\\d+)",
-                        "url-template": "https://static.rust-lang.org/dist/rust-$version-armv7-unknown-linux-gnueabihf.tar.gz"
+                        "type": "rust",
+                        "channel": "stable",
```

Comment:
The channel is set to stable by default

Reply:
I know, just thought that it's better to set it explicitly than imply.